webpack/assets/javascripts/react_app/components/HostDetails/Tabs/ReportsTab/helpers.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable camelcase */
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Button,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  FlexItem,
+  Flex,
+} from '@patternfly/react-core';
+import {
+  ExclamationCircleIcon,
+  SyncAltIcon,
+  CheckCircleIcon,
+  PendingIcon,
+  AngleDoubleRightIcon,
+} from '@patternfly/react-icons';
+import { openConfirmModal } from '../../../ConfirmModal';
+import { APIActions } from '../../../../redux/API';
+import { translate as __ } from '../../../../common/I18n';
+import RelativeDateTime from '../../../common/dates/RelativeDateTime';
+import PuppetPng from '../../../../../../images/Puppet.png';
+import AnsiblePng from '../../../../../../images/Ansible.png';
+
```

Comment:
Can these logos be reused from `app/assets/images/`?

Reply:
done :)